roles/edpm_ovn/tasks/cleanup.yml

```diff
@@ -29,3 +29,11 @@
   changed_when: cleanup_enable_chassis_as_gw.rc == 0
   failed_when: cleanup_enable_chassis_as_gw.rc != 0
   when: not edpm_enable_chassis_gw | bool
+
+- name: Cleanup enable-chassis-as-extport-host when chassis extport not enabled
+  ansible.builtin.shell: >
+    ovs-vsctl remove open . external_ids ovn-cms-options
```

Comment:
So either edpm_enable_chassis_gw and edpm_enable_chassis_extport false will remove ovn-cms-options even one when one of them is true, shouldn't we only remove just the related bit not ovn-cms-options fully?

Reply:
Ah, indeed! Great point. Neither this nor the enable-chassis-as-gw should just get rid of the whole ovn-cms-options IMHO. Because things like AZs also make use of it. I will see what can be done.